docs/howtos/installing_pyenv.md

```diff
@@ -0,0 +1,127 @@
+### Prerequisites
+
+[Git](https://git-scm.com/) installed.
+
+### Install
+
+First check to see if you already have `pyenv` installed by running this in a terminal window:
+
+```
+pyenv
+```
+
+If it is already installed, either update it using `pyenv update` or using the package manager that you used to install it.
+
+If it is not installed you can install it using the following command, pasted into a new terminal window:
+
+```
+curl https://pyenv.run | bash
+```
+
+The output of the command tells you to add certain lines to your startup files for your terminal sessions. Follow the [PyEnv setup instructions](https://github.com/pyenv/pyenv/blob/master/README.md#set-up-your-shell-environment-for-pyenv) copied below - if you are unsure which section to follow, you are probably using a **bash** shell.
+
+  - For **bash**:
+
+    Stock Bash startup files vary widely between distributions in which of them source
+    which, under what circumstances, in what order and what additional configuration they perform.
```

Comment:
I know you did not write this, but I can't help but feel 😵

Reply:
I have tried to rephrase it slightly to be easier to parse.